.ci/configure_podman_for_kata.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+source  "${cidir}/../lib/common.bash"
+
+default_kata_config="/usr/share/defaults/kata-containers/configuration.toml"
+kata_config="/etc/kata-containers/configuration.toml"
+TEST_INITRD="${TEST_INITRD:-no}"
+
+setup_prerequisites() {
+	# Verify host kernel version
+	host_kernel_version=$(uname -r|cut -d. -f1-2)
+	result=$(echo "${host_kernel_version} >= 4.14"|bc)
+	[ "${result}" -ne 1 ] && die "Host kernel version too old"
```

Comment:
Nit: Might be worth logging `${host_kernel_version}` and `4.14` (defined in a variable) in the `die` message for clarity when running under the CI.